pandas/_libs/algos.pyx

```diff
@@ -325,7 +330,7 @@ def nancorr(ndarray[float64_t, ndim=2] mat, bint cov=0, minp=None):
                             sumxx += vx * vx
                             sumyy += vy * vy
 
-                    divisor = (nobs - 1.0) if cov else sqrt(sumxx * sumyy)
```

Comment:
what about ddofv >= nobs case? can you add a test for this. looks like divisor == 0 is handled, but what about ddofv > nobs?